### PR TITLE
docs: Docs update

### PR DESCRIPTION
**Pull request by cartyc**
Updates the Landing Zone v2 documentation. Removed directions for kpt and oci installs to keep things neater and less clustered, I intend to add the back as separate README's. Also added some extra context around some instructions and tweaked the table of contents.

### CODE REVIEW

**Comment by fmichaelobrien** [comment]
Reviewing with a redeploy...

File: docs/landing-zone-v2/README.md [hunk 1]
```diff
@@ -78,13 +69,13 @@ To deploy this Landing Zone you will need to:
 
   1. [Complete the bootstrap procedure.](#1-complete-the-bootstrap-procedure)
   2. [Create your landing zone.](#2-create-your-landing-zone)
-  3. [Deploy the infrastructure using either kpt or gitops-git or gitops-oci.](#3-deploy-the-infrastructure-using-either-kpt-or-gitops-git-or-gitops-oci)
+  3. [Deploy the infrastructure using either gitops](#deploy-the-infrastructure-using-gitops)
```
Comment (davelanglois-ssc) [inline docs/landing-zone-v2/README.md:72]: the word "either" is not required

**Comment by davelanglois-ssc** [comment]
good job chris ! simpler is always better

File: docs/landing-zone-v2/README.md [hunk 2]
```diff
@@ -19,29 +19,20 @@
...
+  - [Create a new repository in your Repo Hosting Solution (Github, Gitlab or Azure Devops)](#create-a-new-repository-in-your-repo-hosting-solution-github-gitlab-or-azure-devops)
+  - [ConfigSync](#configsync)
+- [Validate the landing zone deployment](#4-validate-the-landing-zone-deployment)
+- [Perform the post-deployment steps](#5-perform-the-post-deployment-steps)
+- [Grant billing account user role](#grant-billing-account-user-role)
+- [Next Step](#next-step)
+- [Clean Up](#clean-up)
+  - [Gitops](#clean-up-gitops)
 
 <!-- vscode-markdown-toc-config
```
[the remark this answers is not in the document]
Comment (davelanglois-ssc) [reply docs/landing-zone-v2/README.md:34]: you can look at this file as an example TOC that has been generated with markdown all in one
https://github.com/ssc-spc-ccoe-cei/gcp-documentation/blob/main/Landing%20Zone%20Operations/Building.md

File: docs/landing-zone-v2/README.md (hunk 1 above)
Comment (cartyc) [reply docs/landing-zone-v2/README.md:72]: Will get that fixed this morning. Thanks for the review Dave!

File: docs/landing-zone-v2/README.md [hunk 3]
```diff
@@ -140,10 +131,10 @@ To deploy this Landing Zone you will need to:
 
     ```shell
     # Validate if a storageLocation is defined for the organization
-    gcloud alpha logging settings describe --organization=${ORG_ID}
+    gcloud logging settings describe --organization=${ORG_ID}
```
[the remark this answers is not in the document]
Comment (cartyc) [reply docs/landing-zone-v2/README.md:134]: Thanks! I checked a few other commands that used alpha or beta flags and adjusted where I could. Sadly the billing id assignment was still beta.

File: docs/landing-zone-v2/README.md [hunk 4]
```diff
@@ -67,7 +58,7 @@ This Landing Zone v2 assumes that the different required environments known as E
 ## <a name='SingleGCPorganization'></a>Single GCP organization
 
 ![img](img/single-org.png)
-
+kpt or
```
[the remark this answers is not in the document]
Comment (cartyc) [reply docs/landing-zone-v2/README.md:61]: Horrible typo.

**Comment by cartyc** [comment]
Thanks Dave! The kpt documention always caused some issues with the gatekeeper chicken and egg situation with `constraints` and `constrainttemplates`. I'll move these alternate instructions to their own documents for people to have other options but keep it clear the GitOps method is our preferred and tested option.

**Review by fmichaelobrien — COMMENTED** [review]
worked up to line 415 - doing the set_up_gitops link instead of the removed oci section at 514
reran setup-kcc.sh which automates from line 200-358 

Note: the set_up_gitops section in https://cloud.google.com/anthos-config-management/docs/how-to/config-controller-setup is no longer there or in the config-controller-setup.  We may still need the AR and SA setup in lines 514+

verifying before our meet tomorrow

**Review by fmichaelobrien — APPROVED** [review]
LGTM
These changes are good and required
If we need adjustments we can iterate this and the other doc locations granularly as we go - for expediency and continuous usage by LZ enthusiast deployers

File: docs/landing-zone-v2/README.md [hunk 5]
```diff
@@ -54,11 +56,41 @@ You may want to look at the [documentation](https://github.com/ssc-spc-ccoe-cei/
 ## <a name='Organization'></a>Organization
 
 This Landing Zone v2 assumes that the different required environments known as Experimentation, Development, PreProduction and Production are all instantiated with their own landing zone. This can be achieved with a single GCP organization and multiple landing zone folders representing the environments OR by using multiple GCP organizations.
+- [KRM Landing Zone v2](#krm-landing-zone-v2)
```
Comment (davelanglois-ssc) [inline docs/landing-zone-v2/README.md:59]: double TOC ?

File: docs/landing-zone-v2/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (davelanglois-ssc) [reply docs/landing-zone-v2/README.md:36]: you can remove the entire block. and the one at the top of the TOC. They are not required with markdown-all-in-one
```
<!-- vscode-markdown-toc-config
	numbering=true
	autoSave=true
	/vscode-markdown-toc-config -->
<!-- /vscode-markdown-toc -->
```

**Comment by fmichaelobrien** [comment]
general comment on kpt
I recommend we keep the kpt  and ar docs for now - we can re-add them back in another form later
There are instances where a kpt live apply | destroy are needed - I am thinking dev / standalone mode

I'll have some patches ready starting with the following as I am triaging the core-landing-zone
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/issues/408

**Comment by cartyc** [comment]
I agree that we should keep them, I was think just putting them in a separate document and linking to it from the core instructions. Ideally the core document is our preffered opinionated option and just have `kpt` and `oci` off to the side to reduce the number of decisions users need to make at the start. Something like `documents/landing-zone-v2/alternate-installation.md`.

**Comment by cartyc** [comment]
Excellent, thanks Mike!

**Review by fmichaelobrien — DISMISSED** [review]
LGTM

File: docs/landing-zone-v2/README.md [hunk 6]
```diff
@@ -329,14 +329,16 @@ To deploy this Landing Zone you will need to:
...
+
 1. Get Kubernetes Credentials
 
     ```shell
     gcloud anthos config controller get-credentials $CLUSTER  --location $REGION
     kubens config-control
     ```
 
-1. Set permissions for "Yakima" (Google managed) service account
+1. Set permissions for the "Yakima" (Google managed) service account. This is account is autogenerated with the creation of the Config Controller instance.
```
[the remark this answers is not in the document]
Comment (fmichaelobrien) [reply docs/landing-zone-v2/README.md:344]: This is account is autogenerated with the creation of the Config Controller instance
to
This is the account that is autogenerated with the creation of the Config Controller instance

[the remark this answers is not in the document]
Comment (cartyc) [reply docs/landing-zone-v2/README.md:338]: fixed

File: docs/landing-zone-v2/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cartyc) [reply docs/landing-zone-v2/README.md]: Removed

**Review by fmichaelobrien — APPROVED** [review]
LGTM
- we can continue to iterate

**Pull request merged**